apps/ledger-live-mobile/fastlane/Fastfile

```diff
@@ -151,7 +163,10 @@ platform :ios do
 
       build_number = latest_testflight_build_number(
         version: trim_version_number(package["version"]),
-        app_identifier: "com.ledger.live"
+        # we are moving to using ENV["APP_IDENTIFIER"] but staging + prerelease still override their own
+        # app identifier to publish to com.ledger.live . This will be removed once we are are fully publishing
+        # to separate apps
+        app_identifier: options[:nightly] ? ENV["APP_IDENTIFIER"] : "com.ledger.live"
```

Comment:
can we just put `ENV["APP_IDENTIFIER"]` here , and rely on their values in 
![image](https://github.com/user-attachments/assets/45b15023-d6a0-497c-ae9c-5a260611f901)

Reply:
Yes, this is definitely what I prefer and what I want to get to eventually. The problem is staging still:
- relies on a staging-specific provisioning profile, and it fetches the app identifier from the env file to locate this
- expects to push to the main app under com.ledger.live

So we have to have a hardcoded overwrite somewhere (this is what existed before this PR and I am simply reinstating here, but much more clearly) OR create a new env variable for `APP_IDENTIFIER_FOR_PROVISIONING_PROFILE_LOOKUP_ONLY`. The latter is also not a good long term solution IMO, and we are probably about to send staging builds to one of the new apps (e.g. Prerelease) in an upcoming PR so I don't view the code above as the final solution.

Reply:
ok ! we keep in mind these point for futur improvements.